modules/openxr/openxr_api.cpp

```diff
@@ -562,7 +559,7 @@ bool OpenXRAPI::create_instance() {
 
 	// Create our OpenXR instance
 	XrApplicationInfo application_info{
-		"", // applicationName, we'll set this down below
+		"Godot Game Engine", // applicationName, if we're running a game we'll update this down below.
 		1, // applicationVersion, we don't currently have this
 		"Godot Game Engine", // engineName
```

Comment:
Why are we using `Godot Game Engine` and not just `Godot Engine` here (and for `engineName`)?
The official name of the engine is "Godot Engine".

Reply:
Fixed!